File: Client/src/app/@common/models/employee.ts
import { BaseModel } from '@models/base-model';

export class Employee extends BaseModel {
  // ======================================= //
  public firstName: string;
  public lastName : string;
  public gender   : string;
  public picture  : string;
  // ======================================= //
  constructor(id: number, description: string, title: string, created: Date, firstName: string, lastName: string, gender: string, picture: string) {
    super(id, description, title, created);
    this.firstName = firstName;
    this.lastName  = lastName ;
    this.gender    = gender   ;
    this.picture   = picture  ;
  }
  // ======================================= //
}

File: Shared/Models/Employee.cs
using System.Collections.Generic;

namespace Shared.Models
{
    public class Employee : BaseModel
    {
        // ======================================= //
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Gender { get; set; }
        public string Picture { get; set; }
        // ======================================= //
        //        Navigation Properties
        // ======================================= //
        public virtual IEnumerable<EmployeeTask> Tasks { get; set; }
        // ======================================= //
        public Employee() : base() { }
        public Employee(string firstName, string lastName, string gender, string picture, string title, string description) : base(title, description)
        {
            this.FirstName = firstName;
            this.LastName = lastName;
            this.Gender = gender;
            this.Picture = picture;
        }
    }
}